manifests/charts/gateways/istio-egress/templates/_affinity.tpl

```diff
@@ -70,6 +70,13 @@ nodeAffinity:
           {{- end }}
           {{- end }}
       topologyKey: {{ $item.topologyKey }}
+      {{- if $item.namespaces }}
+      namespaces:
+      {{- $ns := split "," $item.namespaces }}
+      {{- range $i, $n := $ns }}
+      - {{ $n | quote }}
+      {{- end }}
+      {{- end }}
```

Comment:
Is the `namespaces` in affinity added after k8s 1.17?

Reply:
I think it was around from k8s 1.14.

Reply:
Here is the ref:
https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#podaffinityterm-v1-core